tracer/src/Datadog.Trace/Ci/CIEnvironmentValues.cs

```diff
@@ -849,6 +849,16 @@ private void CleanBranchAndTag()
             {
                 Log.Warning(ex, "Error fixing branch name: {BranchName}", Branch);
             }
+
+            if (string.IsNullOrEmpty(Tag))
+            {
+                Tag = null;
+            }
+
+            if (string.IsNullOrEmpty(Branch))
+            {
+                Branch = null;
+            }
```

Comment:
Is this right? This is just trying to make sure that `""` values are set to `null`?

Reply:
yes, this is requried by the change from `SetTag` (where I was avoiding setting the tags on empty string or null) to a custom `ITags` class.